<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KRGSubeBilgileriEkle</class>
 <widget class="QDialog" name="KRGSubeBilgileriEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Şube Ekle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblSubeEkle">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Şube Ekle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSube">
       <property name="title">
        <string>Şube</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
            <item>
             <widget class="QLabel" name="lblSubeAdi">
              <property name="text">
               <string>Şube Adı</string>
              </property>
              <property name="buddy">
               <cstring>lineEditSubeAdi</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSubeAdi"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
            <item>
             <widget class="QLabel" name="lblSubeTelefonNumarasi">
              <property name="text">
               <string>Telefon Numarası</string>
              </property>
              <property name="buddy">
               <cstring>lineEditSubeTelefonNumarasi</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSubeTelefonNumarasi">
              <property name="inputMask">
               <string>\0 (D99) 999 99 99</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
            <item>
             <widget class="QLabel" name="lblSubeYetkilisi">
              <property name="text">
               <string>Şube Yetkilisi</string>
              </property>
              <property name="buddy">
               <cstring>lineEditSubeYetkilisi</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSubeYetkilisi"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Şube İli</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Adana</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Adıyaman</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Afyonkarahisar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ağrı</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Amasya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ankara</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Antalya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Artvin</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Aydın</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Balıkesir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bilecik</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bingöl</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bitlis</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bolu</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Burdur</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bursa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Çanakkale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Çankırı</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Çorum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Denizli</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Diyarbakır</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Edirne</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Elazığ</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Erzincan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Erzurum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Eskişehir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gaziantep</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Giresun</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gümüşhane</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hakkari</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hatay</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Isparta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mersin</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>İstanbul</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>İzmir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kars</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kastamonu</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kayseri</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kırklareli</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kırşehir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kocaeli</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Konya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kütahya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Malatya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Manisa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kahraman Maraş</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mardin</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Muğla</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Muş</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nevşehir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Niğde</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ordu</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rize</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sakarya</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Samsun</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Siirt</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sinop</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sivas</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tekirdağ</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tokat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Trabzon</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tunceli</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Şanlıurfa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Uşak</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Van </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yozgat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Zonguldak</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Aksaray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bayburt</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Karaman</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kırıkkale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Batman</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Şırnak</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bartın</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ardahan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Iğdır</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yalova</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Karabük</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kilis</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Osmaniye</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Düzce</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
            <item>
             <widget class="QLabel" name="lblSubeAdresi">
              <property name="text">
               <string>Şube Adresi</string>
              </property>
              <property name="buddy">
               <cstring>plainTextEditSubeAdresi</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEditSubeAdresi"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="6,1,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonIptal">
         <property name="text">
          <string>İptal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonKaydet">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonKaydet</sender>
   <signal>clicked()</signal>
   <receiver>KRGSubeBilgileriEkle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonIptal</sender>
   <signal>clicked()</signal>
   <receiver>KRGSubeBilgileriEkle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
